ray-operator/config/samples/ray-tpu-scripts/tpu-print-resources.py

```diff
@@ -0,0 +1,19 @@
+import ray
```

Comment:
tpu_print_resources.py or tpu_list_devices.py

Reply:
Done in c11d530.